src/Umbraco.Web/Models/ContentEditing/ContentPropertyDisplay.cs

```diff
@@ -35,5 +35,8 @@ public ContentPropertyDisplay()
 
         [DataMember(Name = "validation")]
         public PropertyTypeValidation Validation { get; set; }
+
+        [DataMember(Name = "linkText")]
+        public string LinkText { get; set; }

```

Comment:
This isn't quite right, we can't be adding more properties to this generic model simply to provide more properties for custom property editors. You can use the 'config' (pre-values) collection to configure a property type which should be used instead.

I am reviewing and modifying this PR so no need to take any actions.